service-wcs/src/main/java/org/oskari/wcs/util/XML.java

```diff
@@ -0,0 +1,94 @@
+package org.oskari.wcs.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.xml.sax.SAXException;
+
+public class XML {
```

Comment:
This class feels like it should be in that no-dependencies util-module that I mentioned in a comment above.